package.json

```diff
@@ -73,7 +73,7 @@
     "fast-glob": "^3.2.7",
     "hardhat": "https://github.com/jflatow/hardhat/releases/download/viaIR/hardhat-v2.9.9.tgz",
     "hardhat-contract-sizer": "^2.4.0",
-    "hardhat-cover": "compound-finance/hardhat-cover",
+    "hardhat-cover": "compound-finance/hardhat-cover#master",
```

Comment:
Isn't this redundant? Isn't it better to just rely on whatever default branch is (i.e. master)

Reply:
yep, no need to update the `package.json`. used `yarn hardhat upgrade hardat-cover` instead to just bump the lock file